src/tutorial/css/index.css

```diff
@@ -0,0 +1,133 @@
+@font-face {
+    src: local('FiraSansOT'),
+         local('FiraSans'),
+         local('Fira Sans OT'),
+         local('Fira Sans'),
+         url('../../media/fonts/FiraSans/firasansot-regular-webfont.woff') format('woff'),
+         url('../../media/fonts/FiraSans/firasansot-regular-webfont.ttf') format('truetype'),
+         url('../../media/fonts/FiraSans/firasansot-regular-webfont.svg#fira_sans_otregular') format('svg');
+    font-family: FiraSansWeb;
+    font-weight: 400;
+}
+
+html {
+    font-family: "Fira Sans OT", "Fira Sans", FiraSansWeb, sans-serif;
+    font-size: 10px;
+}
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+
+    background-color: #000;
+}
+
+#tutorial-container {
```

Comment:
not a fan of id selectors since they have high specificity and are tricky to override